content-src/components/TopSites/_TopSites.scss

```diff
@@ -9,6 +9,7 @@ $default-icon-size: 32px;
 $default-icon-offset: 6px;
 $half-base-gutter: $base-gutter / 2;
 $hover-transition-duration: 150ms;
+$default-link-color: #1A0DAB;
```

Comment:
In order to support dark theme, you'll want to use `var(--newtab-link-primary-color)` here. You could also consider creating a generic link-like-button class you could use here if you want

Reply:
I'll see how often it comes up, and create a class if it keeps happening 👍

Reply:
There's also a bit of padding you need to deal with

Reply:
Our new schmancy component is integrated and seems to be working as expected :)